Poker.BE/Poker.BE.Domain.Tests/Game/RoomTests.cs

```diff
@@ -30,5 +32,78 @@ public void RoomTest()
             TestContext.WriteLine("end of roomTest");
         }
 
+        [TestMethod()]
+        //[ExpectedException(typeof(NotEnoughPlayersException))]
```

Comment:
@idanizi there is 2 way to do it, use expected (like this comment) or with try and catch. Nunit support assert.Throw but not vstest